src/entity/column.rs

```diff
@@ -1198,20 +1193,15 @@ mod tests {
 
                 fn select_as(&self, expr: Expr) -> SimpleExpr {
                     match self {
-                        Self::Two => SimpleExpr::cast_as(
-                            Into::<SimpleExpr>::into(expr),
-                            Alias::new("integer"),
-                        ),
+                        Self::Two => expr.cast_as(Alias::new("integer")),
                         _ => self.select_enum_as(expr),
                     }
                 }
 
-                fn save_as(&self, expr: Expr) -> SimpleExpr {
+                fn save_as(&self, val: Expr) -> SimpleExpr {
```

Comment:
Why change it to val?

Reply:
The rename won't affect the implementation on downstream

Reply:
🫡